library/src/shared/services/auth/auth.service.ts

```diff
@@ -5,6 +5,15 @@ import { ErrorService } from '../error/error.service';
 import jwt_decode, { JwtPayload } from 'jwt-decode';
 import { Constants } from '@constants';
 
+interface CybridJWTPayload extends JwtPayload {
+  sub_type: SubType;
+}
+
+enum SubType {
+  Bank = 'bank',
+  Customer = 'customer'
+}
+
```

Comment:
If we have a model for tokens this should go in there.

Reply:
We don't for this case. I've pulled the decoded JWT model out to it's own model.